docs/layouts/shortcodes/deprecated-versions.md

```diff
@@ -2,5 +2,5 @@
 
 The following versions are [deprecated](/other/glossary.html#deprecated-versions).
 They're available, but they aren't receiving security updates from upstream and aren't guaranteed to work.
-They'll be removed at some point in the future,
-so you should migrate to one of the [supported versions](#supported-versions).
+To avoid issues when they are removed,
```

Comment:
:question: I know there's a general rule to avoid the future, but this is one of the times when I think it makes sense. In the original it's more strongly saying that it will be removed at some point and I think that is important. Is there something here other than trying to avoid the future? If not, I'd revert this change. If so, I'd still include the definitive statement about removal in the future somehow.